.travis.yml

```diff
@@ -130,3 +130,10 @@ notifications:
     skip_join: true
   email:
     - pytest-commit@python.org
+
+branches:
+  only:
+    - master
+    - features
+    - 4.6-maintenance
+    - /^\d+(\.\d)+$/
```

Comment:
Confusing that tags are filtered along with branches, but so it is!

Did you mean to write `/^\d+(\.\d+)+$/` (add `+` to the second `\d`)?

Reply:
Oh, sure.  Amended - thanks for catching!